src/NServiceBus.Core/Transports/Msmq/MsmqConstants.cs

```diff
@@ -0,0 +1,11 @@
+namespace NServiceBus.Transports.Msmq
+{
+    using System.Security.Principal;
+
+    static class MsmqConstants
```

Comment:
Nitpicker: If we are not really reusing these constants I somehow agree with @SimonCropp that we could add it back to the `MsmqQueueCreator`.